tools/scripts/cicd/logging_helper.py

```diff
@@ -0,0 +1,55 @@
+import logging
+from pathlib import Path
+from time import time
+import os
+import sys
+
+
+class LoggingHelper:
+
+    LOG_DIR = "sys_logs"
+    CRITICAL = 50
+    ERROR = 40
+    WARNING = 30
+    INFO = 20
+    DEBUG = 10
+
+    def __init__(self, sub_dir: str, name: str, stdout: bool = False):
+        home = Path.home()
+        directory = str(home) + "/" + self.LOG_DIR + "/" + sub_dir
+        os.makedirs(directory, mode=0o777, exist_ok=True)
+        filename = directory + "/" + name + "_" + str(int(time())) + ".log"
```

Comment:
Thanks for adding time to the log file name. 👍

On the machine right now I see these files: `hil_test_1735594072.log`, `hil_test_1735669825.log`, `hil_test_1735670427.log`, `hil_test_1735671351.log`. I'm imagining seeing something wrong and wanting to find the right log file. Maybe to make debugging easier and faster, I'd suggest also adding short git sha of the commit we're testing to the file name after the timestamp. Would that be reasonably easy? If not in the filename, then near the top of the log file content could be good too.

Reply:
That is a real good idea I will look at adding it!